Jenkinsfile

```diff
@@ -65,5 +65,29 @@ pipeline {
         }
       }
     }
+    stage ('Publish Packages') {
+      when {
+        expression { env.BRANCH_NAME == 'latest' }
+      }
+      agent {
+        docker {
+          image "${nodeImage}"
+          label nodeName
+          args '-u root -v /etc/pki:/certs'
+        }
+      }
+      steps {
+        withCredentials([text(credentialsId: 'something_something', variable: 'PSAMMEAD_NPM_TOKEN')]) {
```

Comment:
I'd suggest just adding these two lines to the existing stage, since the condition (when expression), agent and post-script are identical. Doing this will make the pipeline faster to run.

Reply:
Sorry not 100% sure what you mean, are you saying add
```
withCredentials([text(credentialsId: 'something_something', variable: 'PSAMMEAD_NPM_TOKEN')]) {
  sh 'make publish'
}
```
As a step in the 'Deploy Storybook' stage?